src/App.js

```diff
@@ -1,29 +1,69 @@
 import './App.css';
+import '../node_modules/bootstrap-icons/font/bootstrap-icons.css'
+import {useState} from 'react'
+import {render} from 'react-dom'
+let list=[];
+const input=document.getElementById('input');
+
+let btnFunction='new';
+let clickAction={
```

Comment:
When you think of actions as some global reusable actions, you have to associate that idea with state management, and implement those actions using the state management library of your picking.
Other than that, you should treat your action as component-specific-behavior, in other words, they should exist inside your component.